### DISCUSSION TITLE
1.11.0-alpha

### DISCUSSION THREAD

**Discussion by SkalskiP** [SkalskiP/make-sense#302] [Announcements]
# [1.11.0-alpha] - 2022-12-20

## 🚀 Added

- VOC XML import (https://github.com/SkalskiP/make-sense/pull/280). Now, we can both import and export annotations in VOC XML format.
- Support for Roboflow inference server (https://github.com/SkalskiP/make-sense/pull/298). Now, you can use Roboflow API to annotate images in MS.

https://user-images.githubusercontent.com/26109316/208692860-77fb999e-dd3b-4e23-9fca-c3cfa4751dd5.mp4

## 🔧 Fixed

- Fix support for YOLOv5 P6 models (https://github.com/SkalskiP/make-sense/issues/281). We have resolved problems with loading YOLOv5 M6 models into MS. These were related to the required higher inference resolution.
- Fix confusion around UI for uploading labels (https://github.com/SkalskiP/make-sense/issues/282). UI fixes to make it easier for users to use YOLOv5 models in MS. 
- Fix  non unique label names handling (https://github.com/SkalskiP/make-sense/issues/286). So far, MS has not allowed the user to have multiple labels with the same name, but has not explained in any way why it refuses to load the project. Now an appropriate message with instructions is displayed.
- Fix docker build fails (https://github.com/SkalskiP/make-sense/issues/289). After JS dependencies changes, it was necessary to bump up the required version of the base node image.

## 🏆 Contributors

@hartmannr76, @PawelPeczek, @SkalskiP 

## 🚧 Work in progress

- Our work on [stats view](https://github.com/SkalskiP/make-sense/issues/76) is almost done and you can expect it in the next release. 
- We are working on integration with [MS Inference Server](https://github.com/SkalskiP/make-sense-inference) developed courtesy of @PawelPeczek.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkalskiP/make-sense/releases/tag/1.11.0-alpha'>1.11.0-alpha</a>.</em>

**Comment by undersell** [comment]
Can we copy and paste the bounding box?

**Reply by SkalskiP** [reply]
Hi, @undersell 👋🏻! Thanks for your interest in MS. Unfortunately, not yet. Is that feature that you are looking for?
